modules/rdma-configuring-the-nvidia-network-operator.adoc

```diff
@@ -0,0 +1,230 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-configuring-the-nvidia-network-operator_{context}"]
+
+= Configuring the NVIDIA network Operator
+
+
```

Comment:
Can you add a short intro/abstract here?

Reply:
Added, "The NVIDIA network Operator manages NVIDIA networking resources and networking related components such as drivers and device plugins to enable NVIDIA GPUDirect RDMA workloads."